matrix/utils.py

```diff
@@ -227,6 +227,8 @@ def translate_ansi_colors(entity):
 
     '''
     log.info("Running crash dump")
+    if controller:
+        model_name = "{}:{}".format(controller, model_name)
```

Comment:
@kwmonroe This is a good point. matrix doesn't accept model names prefixed with a controller name, so I think that we're pretty safe here right now.

It is something that might break in the future, though.